src/main/java/novamachina/exnihilosequentia/common/utility/Config.java

```diff
@@ -337,6 +325,8 @@ private static void crucibleConfigs() {
     private static void debugConfigs() {
         enableDebugLogging = COMMON_BUILDER.comment("Enable extra logging? (Default: false)")
                 .define("enableDebugLogging", false);
+        enableStoneblock = COMMON_BUILDER.comment("Enable Stoneblock settings? (Default: false)")
+                .define("enableStoneblock", false);
```

Comment:
This is a little vague for what the config does. Can we get some more info here?

Reply:
will be removed in other pr. moving to own mod